buildbot/osuosl/master/config/builders.py

```diff
@@ -1962,6 +1962,16 @@
                     script_interpreter=None
                 )},
 
+    {'name' : "amdgpu-offload-rhel-8-cmake-build-only",
+    'tags'  : ["amdgpu", "offload", "openmp"],
+    'workernames' : ["rocm-docker-rhel-8"],
+    'builddir': "bbot-build",
```

Comment:
Personally I'd put "amdgpu-offload-rhel-8-cmake-build-only" here to match the builder name, but what you have shouldn't be a problem given that the two builders will never build on the same worker.